python/feathub/processors/local/ast_evaluator/local_ast_evaluator.py

```diff
@@ -152,10 +154,18 @@ def eval_group_node(self, ast: GroupNode, variables: Optional[Dict]) -> Any:
         return self.eval(ast.child, variables)
 
     def eval_is_op(self, ast: IsOp, variables: Optional[Dict]) -> Any:
-        raise RuntimeError("IS/IS NOT operation is not supported.")
+        left_value = self.eval(ast.left_child, variables)
+        return left_value is None or np.isnan(left_value)
 
     def eval_null_node(self, ast: NullNode, variables: Optional[Dict]) -> Any:
-        raise RuntimeError("NULL operation is not supported.")
+        return None
 
     def eval_case_op(self, ast: CaseOp, variables: Optional[Dict]) -> Any:
-        raise RuntimeError("CASE operation is not supported.")
+        for condition_ast, result_ast in zip(ast.conditions, ast.results):
+            if self.eval(condition_ast, variables) is True:
```

Comment:
Let's handle cases in which `self.eval(condition_ast, variables)` returns a value other than `True` and `False`. I found that some sql engines would throw an exception as follows.

> Case error: An expression of non-boolean type specified in a context where a condition is expected